addOns/postman/src/main/resources/org/zaproxy/addon/postman/resources/Messages.properties

```diff
@@ -1,3 +1,11 @@
+postman.api.action.importFile = Imports a Postman collection from a file.
+postman.api.action.importFile.param.endpointUrl = NOT YET IMPLEMENTED.
```

Comment:
Better not.

Reply:
Okay so I can leave them blank, or remove the param for now, which is preferred?

I don't have time to implement the functionality right now.